content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -91,6 +91,27 @@ List of components currently supporting JSON format:
 * {{< glossary_tooltip term_id="kube-scheduler" text="kube-scheduler" >}}
 * {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}
 
+### Log sanitization
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+{{<warning >}}
+Log sanitization might incur significant computation overhead and therefore should not be enabled in production.
+{{< /warning >}}
+
+The `--experimental-logging-sanitization` controls if a sanitization filter will be installed in klog.
+If enabled all log arguments are inspected for fields tagged as sensitive data (e.g. passwords, keys, tokens) and logging of these fields will be prevented.
+
+List of components currently supporting log sanitization:
+* {{< glossary_tooltip term_id="kube-controller-manager" text="kube-controller-manager" >}}
```

Comment:
nit: Line 102, wording
The `--experimental-logging-sanitization` flag enables the `klog` sanitization filter.

nit: It does not make sense to repeat or overuse the tooltips in this section.
Instead you could list the tools.

nit: Wording in note, Line 111:
The Log sanitization filter does not prevent user workload logs from leaking sensitive data.